packages/sanity/src/config/__tests__/resolveConfig.test.ts

```diff
@@ -144,15 +144,16 @@ describe('createWorkspaceFromConfig', () => {
     })
   })
 
-  it('allows overriding the `currentUser` and `client`', async () => {
+  it('allows overriding the `currentUser` and `getClient`', async () => {
```

Comment:
👀 TIL, these are sneaky; not in the interface for config for good reason I presume 😈

Reply:
`createWorkspaceFromConfig` should be considered internal - it is only for testing purposes that you'd want to do this :)